EssentialFeed/Feed API/URLSessionHTTPClient.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public class URLSessionHTTPClient: HTTPClient {
+    struct UnexpectedResultValues: Error {}
```

Comment:
make private

Reply:
rename to something like UnexpectedValuesException to clarify that it is an error